README.md

```diff
@@ -257,6 +257,10 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 // If provided, it MUST be base64url encoded.
                 "nonstandard_id_token_secret": "xxxxxxxxxxxxxxx",
 
+                // Determines if the aud member is required in the
+                // response from the introspection endpoint.
+                "strict_introspection_endpoint_aud": false,
```

Comment:
This will reject access tokens that do not have the `aud` member in the introspection endpoint response.

By extension, by having `aud` be present, we also check that we are part of the `aud`.